pvsite_datamodel/sqlmodels.py

```diff
@@ -264,3 +268,53 @@ class StatusSQL(Base, CreatedMixin):
     status_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
     status = sa.Column(sa.String(255))
     message = sa.Column(sa.String(255))
+
+
+class InverterSQL(Base, CreatedMixin):
+    """Class representing the inverters table.
+
+    Each InverterSQL row represents an inverter identified in
+    a successful Enode link.
+
+    *Approximate size: *
+    4 clients * ~1000 sites each * ~1 inverter each = ~4000 rows
+    """
+
+    __tablename__ = "inverters"
+
+    inverter_uuid = sa.Column(UUID(as_uuid=True), default=uuid.uuid4, primary_key=True)
+    client_uuid = sa.Column(
+        UUID(as_uuid=True),
+        sa.ForeignKey("clients.client_uuid"),
+        nullable=False,
+        comment="The UUID for the client that controls this inverter",
+    )
+    site_uuid = sa.Column(
+        UUID(as_uuid=True),
+        sa.ForeignKey("sites.site_uuid"),
+        nullable=True,
+        comment="The UUID for the site that has this inverter",
+    )
```

Comment:
This table generally maintains a relationship between OCF's DB and Enode's DB. We have a relationship from Enode that a client has many inverters, but we don't know which of OCF's sites these inverters belong to. This table maintains the relationship between the sites table and Enode. We will acquire data from Enode about the inverter as-needed in the API.

Reply:
See my big message in the other thread, but I feel like there should be a 1-1 correspondance between OCF's sites and Enode's inverters!

Reply:
You can have 1 site with 4 inverters. 

I would change the wording from `Enodes inverters` to just `Inverters`. They are physical things, just Enode happens to connect with them

Reply:
I understand that physically there can be many inverters for a given site, but in our setup, a "site" is an all-encompassing term that means "the things to which we associate the time series".

In the end the "site" is only a way to identify a given time series and add some metadata related to the time series. In real life it might very well map to an inverter.

I'm not sure we need the distinction between real-life sites and real-life inverters: in the case were 1 real-life site has 4 inverters, we can always define that as 4 distinct sites in the database. (I'm assuming these 4 inverters each have their own time series.)

In any case we should agree on the definitions because it's already getting confusing: on one hand we are adding an inverter table, but on the other we are adding the inverter capacity to the sites table.